Bottom: e6126246666021dc9ca69219fc0244cbf661bb7a
Top:    ee2cebe1de99b232e8abc181c7cf684ed92e8570
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-20 09:51:00 -0500

Update configuration for running behind reverse proxy



---
diff --git a/roles/buildbot-master/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
index d23f6eee7..656c05a3e 100644
--- a/roles/buildbot-master/defaults/main.yml
+++ b/roles/buildbot-master/defaults/main.yml
@@ -47,7 +47,7 @@ container_env:
   BUILDBOT_SMTP_USER: 'buildbot'
   BUILDBOT_SMTP_PASS: '{{ buildbot_email_password }}'
   BUILDBOT_WEB_PORT: '{{buildbot_http_port}}'
-  BUILDBOT_WEB_URL: 'http://{{ buildbot_hostname }}/'
+  BUILDBOT_WEB_URL: 'https://{{ buildbot_hostname }}/'
   BUILDBOT_WORKER_PORT: '{{buildbot_worker_port}}'
   BUILDBOT_WORKER_NTEL: 'worker-ntel'
   BUILDBOT_WORKER_NTEL_PASS: '{{ buildbot_worker_ntel_password }}'
